deploy/helm/kubecf/assets/operations/instance_groups/database.yaml

```diff
@@ -1,3 +1,323 @@
+- type: remove
+  path: /instance_groups/name=diego-api/jobs/name=silk-controller/properties/database/require_ssl?
+
+- type: replace
+  path: /instance_groups/name=diego-api/jobs/name=locket/properties/diego/locket/sql/db_driver
+  value: {{ .Values.features.external_database.type | quote }}
+- type: replace
+  path: /instance_groups/name=diego-api/jobs/name=locket/properties/diego/locket/sql/db_port
+  value: {{ .Values.features.external_database.port }}
+- type: replace
+  path: /instance_groups/name=diego-api/jobs/name=locket/properties/diego/locket/sql/db_schema
+  value: {{ .Values.features.external_database.databases.locket.name | quote }}
+- type: replace
+  path: /instance_groups/name=diego-api/jobs/name=locket/properties/diego/locket/sql/db_host?
+  value: {{ .Values.features.external_database.host | quote }}
+- type: replace
+  path: /instance_groups/name=diego-api/jobs/name=locket/properties/diego/locket/sql/db_password
+  value: {{ .Values.features.external_database.databases.locket.password | quote }}
+- type: replace
+  path: /instance_groups/name=diego-api/jobs/name=locket/properties/diego/locket/sql/db_username
+  value: {{ .Values.features.external_database.databases.locket.username | quote }}
+- type: remove
+  path: /instance_groups/name=diego-api/jobs/name=locket/properties/diego/locket/sql/ca_cert?
+- type: remove
+  path: /instance_groups/name=diego-api/jobs/name=locket/properties/diego/locket/sql/require_ssl?
+
+- type: replace
+  path: /instance_groups/name=credhub/jobs/name=credhub/properties/credhub/data_storage/type
+  value: {{ .Values.features.external_database.type | quote }}
+- type: replace
+  path: /instance_groups/name=credhub/jobs/name=credhub/properties/credhub/data_storage/port
+  value: {{ .Values.features.external_database.port }}
+- type: replace
+  path: /instance_groups/name=credhub/jobs/name=credhub/properties/credhub/data_storage/database
+  value: {{ .Values.features.external_database.databases.credhub.name | quote }}
+- type: replace
+  path: /instance_groups/name=credhub/jobs/name=credhub/properties/credhub/data_storage/host
+  value: {{ .Values.features.external_database.host | quote }}
+- type: replace
+  path: /instance_groups/name=credhub/jobs/name=credhub/properties/credhub/data_storage/password
+  value: {{ .Values.features.external_database.databases.credhub.password | quote }}
+- type: replace
+  path: /instance_groups/name=credhub/jobs/name=credhub/properties/credhub/data_storage/username
+  value: {{ .Values.features.external_database.databases.credhub.username | quote }}
+- type: remove
+  path: /instance_groups/name=credhub/jobs/name=credhub/properties/credhub/data_storage/tls_ca?
+- type: replace
+  path: /instance_groups/name=credhub/jobs/name=credhub/properties/credhub/data_storage/require_tls?
+  value: false
+
+{{- else -}}
+
+# Override the addresses for the jobs under the scheduler instance group.
```

Comment:
Not sure this should be in this file.

I think, ideally, we'd _not_ change this file, but have an _additional_ ops file to remove it all again (applied after this one).

But that's because I want to have a `…/features/` directory to organize all our features-based things, I think.

Reply:
I generally dislike having the same condition in multiple places of the project. I.e. a single helm value controlling multiple conditions in multiple different files. It's acceptable in some cases that we cannot do it from a single file. We should discuss if the advantage of having `.../features/` would pay off.